### PR TITLE
Video component

### PR DESCRIPTION
**Pull request by cbandrow-zz**
## What's this PR do?
Did this thing. I redid the home page to make user experience more clear. Now complete with background video! Neat-o!

Additionally, added a few tests to the LandingPage component. 

ovirtq6gyp

## Previous UI: 
 screen shot 2017-06-06 at 13 47 41

screen shot 2017-06-06 at 13 48 06

## Updated UI: 
screen shot 2017-06-06 at 13 45 39

screen shot 2017-06-06 at 13 46 00

screen shot 2017-06-06 at 13 46 10

## Learning
The thousands of websites about Videos in HTML 5 and Leta.

## Where should the reviewer start?

Feel free to check out the LandingPage Component as well as its own tests and css. 
Additionally, if you want to run it, clone the repo, npm install, npm run build, npm start. 

## What gif best describes this PR or how it makes you feel?

Also, feel free to ignore the bundle.js... I just realized they weren't in my gitignore :(

### CODE REVIEW

**Comment by Adamj1232** [comment]
Looks great and much more smooth for UX!

**Comment by zanedr** [comment]
UI looks great dude!

**Comment by DylanBohlender** [comment]
4/5, cars are driving on the wrong side of the road

**Comment by rrgayhart** [comment]
that's pretty neat

**Comment by rrgayhart** [comment]
Hooray for removing the DS_Stores!

File: app/assets/styles/main.css [hunk 1]
```diff
@@ -132,6 +168,17 @@ header h1 {
   outline: none;
 }
 
+#uploaded{
```
Comment (rrgayhart) [inline app/assets/styles/main.css:171]: Watch `ids` being not specific enough

File: app/components/App.js [hunk 2]
```diff
@@ -97,11 +98,11 @@ export default class App extends Component {
   displayComponents(){
     if(this.state.imagePreviewUrl){
       return (
-        <div className = 'content-holder'>
-          <ImageHolder url = {this.state.imagePreviewUrl}/>
-          <ResultsHolder cars = {this.state.compareResults}
-          loadingStatus = {this.state.loadingStatus} error = {this.state.error}/>
-        </div>
+          <section className = 'content-holder'>
```
Comment (rrgayhart) [inline app/components/App.js:101]: Killing divs!

**Comment by ghost** [comment]
Looking super clean!

**Pull request merged**